app/javascript/stylesheets/QuestionnaireForm.scss

```diff
@@ -273,3 +273,27 @@ input {
     transition: flex 400ms ease;
   }
 }
+
```

Comment:
Could you put these styles in `questions.scss` instead? `QuestionnaireForm.scss` is for the admin questionnaire builder only (the name is a little misleading, we might change this in the future)